ci/rpm/test_daos_node.sh

```diff
@@ -4,7 +4,10 @@ set -uex
 sudo yum -y install --exclude ompi daos{,-client}-"${DAOS_PKG_VERSION}"
 sudo yum -y history rollback last-1
 sudo yum -y install --exclude ompi daos{,-{server,client}}-"${DAOS_PKG_VERSION}"
-sudo yum -y install --exclude ompi daos{,-tests}-"${DAOS_PKG_VERSION}"
+sudo yum -y install --exclude ompi daos-tests{,-common}-"${DAOS_PKG_VERSION}"
+sudo yum -y install --exclude ompi \
+  daos-tests-{ior,fio,mpiio,hdf5-vol,macsio,soak}-"${DAOS_PKG_VERSION}"
```

Comment:
These should all be install-able in the same `yum` command.

Reply:
There's a little history on this one.  I had tried to use one command with:
```
sudo yum -y install --exclude ompi \
  daos-tests{,-{common,ior,fio,mpiio,hdf5-vol,macsio,soak}}-\
  "${DAOS_PKG_VERSION}"
```
but the assembly of the string included the spaces and caused an error with the command.

I then tried this edit but it exceeded 80 chars:
```
sudo yum -y install --exclude ompi \
  daos-tests{,-{common,ior,fio,mpiio,hdf5-vol,macsio,soak}}-"${DAOS_PKG_VERSION}"
```

So the current implementation was the compromise, but if you have a better solution I'll give it a try.

Reply:
I'd be fine with it being an 80-char limit exception.  But alternatively I think you can do:

```
sudo yum -y install --exclude ompi \
  daos-tests{,-{common,ior,fio,mpiio,hdf5-vol,macsio,soak}}-\
"${DAOS_PKG_VERSION}"
```

Albeit it does mess with the visual representation that the indent provides.